docs/fermi_hubbard/index.md

```diff
@@ -0,0 +1,23 @@
+# Fermi-Hubbard
+
+Strongly correlated quantum systems give rise to many exotic physical phenomena, 
+including high-temperature superconductivity. 
+Simulating these systems on quantum computers may avoid the prohibitively 
+high computational cost incurred in classical approaches. 
+However, systematic errors and decoherence effects presented in current quantum 
+devices make it difficult to achieve this. 
+We simulated the dynamics of the one-dimensional Fermi-Hubbard model 
+using 16 qubits on Google's quantum processor in 
+[arxiv:2010.07965](https://arxiv.org/abs/2010.07965).
+We observed separations in the 
+spreading velocities of charge and spin densities in the highly excited regime, 
+a regime that is beyond the conventional quasiparticle picture. 
+We employed a sequence of error-mitigation techniques to reduce decoherence 
+effects and residual systematic errors. 
+These procedures allowed us to simulate the time evolution of the model 
+faithfully despite having over 600 two-qubit gates in our circuits.
+
+This module contains the code used to generate Fermi-Hubbard circuits,
+execute the experiments, and analyze the results. Please read on to see
+an example experiment and analysis of the published data.
+
```

Comment:
Could you change this to that (your text described our paper more than the code itself):

# Simulation of spin and charge dynamics in the Fermi-Hubbard model

This module contains an implementation of code to simulate the dynamics of the one-dimensional Fermi-Hubbard model on Google's Quantum Computing Service

## Overview

The examples allow to analyze and reproduce the results published in [arXiv:2010.07965](https://arxiv.org/abs/2010.07965) **[quant-ph]**:

  * The [experiment_example.ipynb](fermi_hubbard/experiment_example.ipynb) 
  notebook guides on how the 
  code is structured, how to simulate the Fermi-Hubbard model 
  and execute the experiment using Google's Quantum Computing Service.
  * The [publication_results.ipynb](fermi_hubbard/publication_results.ipynb) 
  notebook guides through 
  the analysis of results from the data set which was released together with the
  publication. The data set consists of four zip files and can be accessed at 
  [https://doi.org/10.5061/dryad.crjdfn32v](
  https://doi.org/10.5061/dryad.crjdfn32v). The files need to be downloaded and
  extracted into the ```docs/fermi_hubbard/fermi_hubbard_data``` directory.

Reply:
done